File: VB/Scaffolding.CodeFirst/Data/Course.vb
Imports System.ComponentModel
Imports System.ComponentModel.DataAnnotations

Namespace Scaffolding.CodeFirst.Model
	Public Class Course
		Public Property CourseID() As Integer
		Public Property Title() As String
		Public Property DepartmentID() As Integer
		Public Overridable Property Department() As Department
	End Class
End Namespace

File: CS/Scaffolding.CodeFirst/Data/Course.cs
using System.ComponentModel;
using System.ComponentModel.DataAnnotations;

namespace Scaffolding.CodeFirst.Model {
    public class Course {
        public int CourseID { get; set; }
        public string Title { get; set; }
        public int DepartmentID { get; set; }
        public virtual Department Department { get; set; }
    }
}